### PR TITLE
enh: changing status route to invoice

### LINKED ISSUE #3212: Change route name from sales/status to sales/invoice
This is a sub issue of #3132 

To be done : Change https://eventyay.com/admin/sales/status to https://eventyay.com/admin/sales/invoices

### PR DESCRIPTION
**Pull request by uds5501**
Fixes #3212 

#### Short description of what this resolves:
Renames sales/status route to sales/invoice

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by uds5501** [comment]
Screen shot of results after the changes :
Screenshot from 2019-06-28 10-09-28

**Comment by uds5501** [comment]
@mrsaicharan1 @shreyanshdwivedi @kushthedude please review

File: app/router.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ router.map(function() {
       this.route('marketer');
       this.route('locations');
       this.route('fees');
-      this.route('status');
+      this.route('invoice');
```
Comment (niranjan94) [inline app/router.js]: Should be `invoices`

Comment (uds5501) [reply app/router.js]: @niranjan94 changed it

File: app/templates/admin/sales.hbs [hunk 2]
```diff
@@ -20,7 +20,7 @@
         {{#link-to 'admin.sales.fees' class='item'}}
           {{t 'Fees'}}
         {{/link-to}}
-        {{#link-to 'admin.sales.status' class='item'}}
+        {{#link-to 'admin.sales.invoice' class='item'}}
```
Comment (Anupam-dagar) [inline app/templates/admin/sales.hbs]: shouldn't it be `invoices` too? I guess the route name and file names must be same.

Comment (uds5501) [reply app/templates/admin/sales.hbs]: @Anupam-dagar you are right. pushing the fix.

**Comment by uds5501** [comment]
@niranjan94 please review this

**Pull request merged**